Technomancy/tower.ts

```diff
@@ -691,9 +702,9 @@ function climb_tower(health = undefined, ehealth = undefined, grinding = false)
             } else if ((attack == "attack" && enemy_attack == "dodge") || (attack == "dodge" && enemy_attack == "spaz") || (attack == "spaz" && enemy_attack == "attack")) {
                 winstate = "lost";
             }
-            let enemy_damage = Math.pow(adventure_data["tower_floor"], 2);
+            let enemy_damage = Math.pow(adventure_data["tower_floor"], 2) * (1 + adventure_data["tower_ascension"]);
             if (grinding) {
-                enemy_damage = Math.pow(grinding_level, 2);
+                enemy_damage = Math.pow(grinding_level, 2) * (1 + adventure_data["tower_ascension"]);
```

Comment:
Hmm... on a balance level, is this too mean? Just ascending puts grinding tower at +4 floors already, making it harder to hit the top, but then also adding difficulty multiplier? Maybe also add the same multiplier to the essence gained per floor then? Seems like something that'll need to be playtested.

Reply:
Or potentially keep it at one ascension behind? So ascensions 0 and 1 are 1x, then 2 is at 2x..., instead of 0 is 1x, 1 is 2x...

Reply:
To answer the first, yes.  I think it is too mean.